### ISSUE TITLE
Enhance filter functionality

### ISSUE THREAD

**Issue by vembry** [rangav/thunder-client-support#419]
**Is your feature request related to a problem? Please describe.**

I'm not sure whether this is a problem or not. But this might ease in searching existing requests.

##
**Describe the solution you'd like**

I would like to have the filter feature to include endpoints in the search. 
Given i have the following requests in my collection
```
request 1
- request name: do something
- endpoint: {host}/alpha/do-something

request 2
- request name: do another thing
- endpoint: {host}/charlie/do-another-thing

request 3
- request name: do something else
- endpoint: {host}/zulu/do-something-else/
```
when I tried to search for "thing", it'll show all 3 of it,
when I search for "zulu", it'll only show `request 3`, since it's endpoints contains the word `zulu`

##
**Describe alternatives you've considered**

i havent considered alternatives

##
**Implementation:**

```javascript
// example thunder client request collections based on thunderclient.db
let thunderClientRequests = [
    {
        //other thunder client db property...
        name: "do something",
        url: "{{host}}/alpha/do-something"
        //other thunder client db property...
    },{
        //other thunder client db property...
        name: "do another thing",
        url: "{{host}}/charlie/do-another-thing"
        //other thunder client db property...
    },{
        //other thunder client db property...
        name: "do something else",
        url: "{{host}}/zulu/do-something-else"
        //other thunder client db property...
    }
]

// keyword search
let filterKeyword = "word"

// results
let filteredThunderClientRequests = thunderClientDb.filter(x => x.url.includes(filterKeyword))
```

p.s pls mention existing issue ticket if one has raise this before.

**Comment by rangav** [comment]
Hi @vembry thanks for the feedback, will look into it.

**Comment by rangav** [comment]
This feature is now implemented in v1.11.0
https://github.com/rangav/thunder-client-support/releases/tag/v1.11.0

please let me know if any feedback
